manifests/install.pp

```diff
@@ -40,9 +40,9 @@
 
   if !$::vault::disable_mlock {
     exec { "setcap cap_ipc_lock=+ep ${vault_bin}":
```

Comment:
My apologies for the noise I caused ... Repeatedly, I was unable to run the tests travis-ci does locally. bundle exec rake spec throws 25 failures where travis completes without any.

Depending on how my workload develops during the week, I will work on #50. Thank you for your work on puppet-vault!